packages/ui/src/machines/authenticator/index.ts

```diff
@@ -338,6 +338,7 @@ export function createAuthenticatorMachine() {
               username: context.actorDoneData?.authAttributes?.username,
               formFields: context.config?.formFields,
               validationError: {},
+              passwordSettings: context.config?.passwordSettings,
```

Comment:
Should this be in be shared with `spawnSignInActor` too? (for force-new-password change)

Reply:
Yes passwordSettings are required for showing validation errors.